src/main/resources/org/jenkinsci/plugins/plaincredentials/impl/FileCredentialsImpl/credentials.jelly

```diff
@@ -35,7 +35,7 @@ THE SOFTWARE.
                 <f:textbox field="fileName"/>
             </f:invisibleEntry>
             <f:invisibleEntry>
-                <f:textbox field="data"/>
+                <f:textbox field="secretBytes"/>
```

Comment:
Perhaps not possible due to `readResolve`, unless the field name deliberately differs from the public getter & constructor parameter names (which is permitted but also confusing).